interface-spec.md

```diff
@@ -99,6 +109,7 @@ TODO: Do we need to define a different interface, or is a quad simply a triple w
 - `.blankNode()` returns a new instance of BlankNode.
 - `.literal(String value, String language, String datatype)` returns a new instance of Literal.
 - `.variable(String name)` returns a new instance of Variable. This method is optional.
+- `.defaultGraph()` returns an instance of DefaultGraph. This method should return a singleton.
```

Comment:
If multiple libraries are used it can't be the same constant.
Am 22.01.2016 10:11 vorm. schrieb Ruben Verborgh <notifications@github.com>:In interface-spec.md:
> @@ -99,6 +109,7 @@ TODO: Do we need to define a different interface, or is a quad simply a triple w
>  - `.blankNode()` returns a new instance of BlankNode.
>  - `.literal(String value, String language, String datatype)` returns a new instance of Literal.
>  - `.variable(String name)` returns a new instance of Variable. This method is optional.
> +- `.defaultGraph()` returns an instance of DefaultGraph. This method should return a singleton.

"This method should return a singleton." => "This method always returns the same constant."

—Reply to this email directly or view it on GitHub.